src/slurmrestd_ops.py

```diff
@@ -111,6 +111,7 @@ class SlurmrestdManager:
     def __init__(self):
         self._munge_package = CharmedHPCPackageLifecycleManager("munge")
         self._slurmrestd_package = CharmedHPCPackageLifecycleManager("slurmrestd")
+        self._slurm_plugins_package = CharmedHPCPackageLifecycleManager("slurm-wlm-basic-plugins")
```

Comment:
[nit]: Remind me, why did we need to include this package specifically? I thought we needed the slurm-client package here instead.

Reply:
Because `slurmrestd` requires `libslurmfull.so`, which is provided by `slurm-wlm-basic-plugins`. We could also include the full `slurm-client`, but we would also be pulling a lot of binaries that we don't really need.

Reply:
I mean, ideally `slurmrestd` would depend on `slurm-wlm-basic-plugins`, but ¯\\\_(ツ)_/¯

Reply:
Hmm... Okay. I remember you mentioning needing to use slurm-client over Mattermost, but weird that the default slurmrestd package doesn't provide everything slurmrestd needs to start.

Seems like a bug with the deb package version that we're pulling 🪳